package common.protocol.G2L;

message S2CGamedataSend
{
	required int32 cmd_type_no = 1;
	required int64 roleid      = 2;
	required int32 link_id     = 3;
	required int32 client_sid_in_link = 4;

	required bytes content     = 5;
}
